Artemis/Artemis/Utilities/ActiveWindowHelper.cs

```diff
@@ -49,6 +59,28 @@ public static class ActiveWindowHelper
         private static void ActiveWindowChanged(IntPtr hWinEventHook, uint eventType,
             IntPtr hwnd, int idObject, int idChild, uint dwEventThread, uint dwmsEventTime)
         {
+            UpdateForWindow(hwnd);
+        }
+
+        private static void WindowTitleChanged(IntPtr hWinEventHook, uint eventType,
+            IntPtr hwnd, int idObject, int idChild, uint dwEventThread, uint dwmsEventTime)
+        {
+            if (_activeWindow == hwnd)
```

Comment:
Yeah debugging shows it works for me and I believe comparing IntPtr just does a value comparison. Since they are both pointing to the same window handler they both have the same value